hearts/tests/test_game.py

```diff
@@ -108,3 +112,12 @@ def test_rankings():
                                     p1: 1,
                                     p2: 2,
                                     p3: 4}
+
+
+def test_deserialize_game_check_player_data():
+    test_players = [Player('Lauren'), Player('Erin'), Player('Jeremy'), Player('Daniel')]
+    test_game, _ = new_game(test_players, points_to_win=27)
+    test_game.players = test_players  # Seat players in the order: Lauren, Erin, Jeremy, Daniel
+    assert test_game.players == test_players
+    print ([player.username for player in test_game.players])
+    assert test_game.players == [Player('Lauren'), Player('Erin'), Player('Jeremy'), Player('Daniel')]
```

Comment:
Yes, you can see in test output where it says "captured standard output"

![screen shot 2017-03-27 at 4 32 22 pm](https://cloud.githubusercontent.com/assets/2467754/24382415/eee3130a-130a-11e7-9cef-739ef3fc0eef.png)

And compare with the test assertion

![screen shot 2017-03-27 at 4 32 27 pm](https://cloud.githubusercontent.com/assets/2467754/24382418/f3ca8c4a-130a-11e7-985f-fdd178a994a8.png)


If you want to fix the test, you can do a set equality:

```
assert set(test_game.players) == set([Player('Lauren'), Player('Erin'), Player('Jeremy'), Player('Daniel')])
```

Unless you specifically want to test the ordering of the players?